apptest/tests/metricsql_test.go

```diff
@@ -173,3 +178,29 @@ func testInstantQueryDoesNotReturnStaleNaNs(t *testing.T, sut apptest.Prometheus
 		t.Errorf("unexpected response (-want, +got):\n%s", diff)
 	}
 }
+
+func testQueryRangeWithAtModifier(t *testing.T, sut apptest.PrometheusWriteQuerier) {
```

Comment:
@zekker6 this is missing the explanation that test would fail only for certain platforms or versions of go.

Reply:
Added a comment to clarify this